precompiles/gov/integration_test.go

```diff
@@ -363,5 +362,118 @@ var _ = Describe("Calling governance precompile from EOA", func() {
 				}
 			})
 		})
+
+		// TODO: Can't add deposit manually without the precompile transaction available ?
```

Comment:
can add the deposit on the custom genesis of the `SetupTest` function in the `setup_test.go` file